broadinstitute/single_cell_portal_core
# Update wizard form UI for AnnData experience (SCP-4880)

This makes the updates to the Upload Wizard for the AnnData upload experience. This is pretty much purely a visual change. It removes and adds the appropriate form fields depending on whether it is in `isAnnDataExperience` or not.

To test:
- Pull this branch 
- Change line 74 `  const [isAnnDataExperience, setIsAnnDataExperience] = useState(false)` to be `  const [isAnnDataExperience, setIsAnnDataExperience] = useState(true)` 
- Start up your local server
- Log in and head to a blank study
- Click to the upload wizard
- You should see the updates to the available forms
- namely:
- Only the 4 main steps (Raw, Processed, Metadata, Clustering), Sequence files, Misc. and Seurat.
- No upload/save/delete/add buttons in any of the 4 main steps
- Metadata only has the image and link to view conventional metadata
- No " ‘matrix file type’ form field" in both Raw and Processed file forms
- ‘associated raw counts files’ form field is removed from Processed 
- Clustering has the addition of the 'OBSM Key names" field *note this name is final, awaiting input from Jean

https://user-images.githubusercontent.com/54322292/214674642-42178bec-a6f7-4f85-bfc6-f27fdfbcd993.mov
